lib/matplotlib/pyplot.py

```diff
@@ -1997,6 +1997,49 @@ def colormaps():
     return sorted(cm.cmap_d)
 
 
+def verbose(level='info'):
```

Comment:
This doesn't have the same default as matplotlib.verbose...
FWIW I would just do `from matplotlib import verbose` and be done with it.

Reply:
will that show up in the `pyplot` docs?

Reply:
Fixed the defaults.